src/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import * as serviceWorker from './serviceWorker';
```

Comment:
try to commit first all the code
and then start work on your code
i dont need to read all the non relevant code either.